src/settings/FeeFineSettings.js

```diff
@@ -317,13 +317,15 @@ class FeeFineSettings extends React.Component {
           onSubmit={this.onUpdateOwner}
           hasEditOwnerPerm={hasEditOwnerPerm}
         />
-        <CopyModal
-          {...this.props}
-          openModal={this.state.showCopyDialog}
-          onCopyFeeFines={this.onCopyFeeFines}
-          onCloseModal={this.hideCopyDialog}
-          ownerList={filterOwners}
-        />
+        <IfPermission perm="feefines.item.post feefines.item.put">
```

Comment:
Should we use a comma to split permission names?
```suggestion
        <IfPermission perm="feefines.item.post,feefines.item.put">
```